src/MainTest.java

```diff
@@ -70,4 +70,51 @@ public void testSetAndGetHeight() {
         Assertions.assertEquals(newHeight, actualHeight);
     }
 
+    @Test
+    public void testCreateSquare(){
+        double side = 4;
+        Main.Square square = new Main.Square(side);
+        Assertions.assertNotEquals(square,null);
+    }
+
+    @Test
+    public void testCalculateSquareArea(){
+        double side = 4;
+        Main.Square square = new Main.Square(side);
+        double expectedArea = 4 * side;
+        Assertions.assertEquals(expectedArea,square.calculateArea());
+    }
+
+    @Test
+    public void testSetSquareWidth(){
+        double firstSide = 4;
+        Main.Square square = new Main.Square(firstSide);
+        double secondSide = 6;
+        square.setWidth(secondSide);
+        Assertions.assertEquals(secondSide,square.getWidth());
+    }
+
+    @Test
+    public void TestWidthAndHeightAreEqual(){
```

Comment:
We don't need this Test

Reply:
we should test if setSide id correctly update both width and height as they should be equal